components/modal/stories/modal.stories.js

```diff
@@ -31,26 +34,44 @@ export default {
 	args: {
 		isOpen: true,
 		rootClass: "spectrum-Modal",
-		// @todo use a more exciting example
-		content: ["Modal is a base component used by other components, and should not be used on its own."],
 	},
 	parameters: {
+		layout: "fullscreen",
+		docs: {
+			story: { height: "600px" },
+		},
 		componentVersion: version,
 	},
 	decorators: [
-		/**
-		 * Ensure modal is is not cropped out of the Chromatic screenshot.
-		 * @see https://www.chromatic.com/docs/snapshots/#why-isn%E2%80%99t-my-modal-or-dialog-captured 
-		 */
-		(story) => window.isChromatic() ? html`<div style="width: 1200px; height: 800px;">${story()}</div>` : story()
+		withUnderlayWrapper,
 	],
 };
 
 export const Default = ModalGroup.bind({});
-Default.args = {};
+Default.args = {
+	content: [
+		(passthroughs, context) => Dialog({
```

Comment:
> We're also missing testing of the variants ("Sizing preference"). And changing this to fullscreen currently does not display correctly with the Dialog

We weren't testing those variants to begin with and the complexity of supporting them didn't make sense based on the goals for this PR. We can add a backlog story to add those later?

Reply:
Adding coverage for sizing preference can definitely be done in a followup. But I don't think we should be displaying and testing Dialog here instead of just Modal.

Reply:
Modal doesn't snapshot well without something inside of it - do you have a suggestion for what to put inside Modal to render it in a way we can test?

Reply:
In #2854 I added an option to display the Underlay like Dialog does. That way the background and rounded corners are visible.